app/abilities/ability.rb

```diff
@@ -71,12 +75,19 @@ def medlem
   def medlem_has_role
     # If the user has the Pages owner role
     can [:admin, :edit, :update, :preview], Page, role_id: @user.sub_roles.pluck(:id)
+  end
+
+  def medlem_has_child_roles
```

Comment:
I don't like that fact that we are switching between `medlem` and `member`. We should be consistent and only use `member`.